calculator.c

```diff
@@ -0,0 +1,168 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+#define max 500
+int oprand[max], vtop = -1, otop = -1;
+char operator[max]; 
+
+void pushvalue(int value)
+{
+	vtop = vtop + 1;
+	oprand[vtop] = value;
+}
+
+int popvalue()
+{
+    int value = oprand[vtop];
+    vtop = vtop - 1;
+    return value;
+}
+
+void pushoprator(char op)
+{
+	otop = otop + 1;
+	operator[otop] = op;
+}
+
+char popoprator()
+{
+    char op = operator[otop];
+    otop = otop - 1;
+    return op;
+}
+
+int isDigit(char c)
+{
+	if(c>='0' && c<='9')
+	{
+		return 1;
+	}
+	else
+	{
+		return 0;
```

Comment:
Avoid multiple return statements in single function. The function should have one entry and one exit.